packages/vhd-lib/Vhd/VhdAbstract.js

```diff
@@ -200,14 +200,6 @@ exports.VhdAbstract = class VhdAbstract {
     }
   }
 
-  static async rename(handler, sourcePath, targetPath) {
-    try {
-      // delete target if it already exists
-      await VhdAbstract.unlink(handler, targetPath)
-    } catch (e) {}
-    await handler.rename(sourcePath, targetPath)
-  }
-
```

Comment:
IMHO, this fix should be in a separate commit, what do you think? :slightly_smiling_face:

Reply:
yes. I kept all the commit separated to eas th review, if the review is ok  , I will squash the first four commits, and rework the last one with a detailled explanation  ( I am sure I will berate past me in the future for removing this )